### PR TITLE
Fixing broken ROOT runtime_cxxmodules build 

### PR DESCRIPTION
**Pull request by oshadura**

### CODE REVIEW

**Comment by vgvassilev** [comment]
@pcanal, @dpiparo could you review and let us know if you have any objections for merging in this patch.

This LGTM and fixes a *huge* regression in `-Druntime_cxxmodules=On`.

**Comment by vgvassilev** [comment]
@phsft-bot build just on slc6/gcc62 with flags -Druntime_cxxmodules=On -Dctest_test_exclude_none=on

File: root/tree/cloning/generate-eventfile.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+ROOT_DIR=$1
+
+${ROOT_DIR}/test/eventexe 6 0 0 1 30 > log
```
Comment (pcanal) [inline root/tree/cloning/generate-eventfile.sh]: Please use a less generic name for the log file ....

**Comment by oshadura** [comment]
@pcanal sorry, I changed name here https://github.com/root-project/roottest/pull/151

**Comment by pcanal** [comment]
thanks

**Pull request merged**